server/src/main/scala/io/delta/kernelsharedtable/JsonPredicates.scala

```diff
@@ -0,0 +1,492 @@
+/*
```

Comment:
Isn't this already available in server/src/main/scala/io/delta/standalone/internal/JsonPredicates.scala?

Reply:
Yeah it seems very similar, let me pull the version from internal into the common folder, remove this one and see what happens

Reply:
done, this worked and I moved the standalone JsonPredicates.scala to common and now the kernel and standalone code uses this same file